File: AOJ/0431/0431.c
#include<stdio.h>

int main(){
  int i;
  int N, r[1000];  /* 1000個の円盤の半径を記録できる配列 */
  int maxr = 0, cnt = 0;
  scanf("%d", &N);
  for ( i = 0; i < N; i++ ) scanf("%d", &r[i]);
  for ( i = N - 1; i >= 0; i-- ){
    if ( r[i] > maxr ) {
      cnt++;
      maxr = r[i];
    }
  }
  printf("%d\n", cnt);
  return 0;
}

File: AOJ/0431/0431.py
N = int(input())
r = list(map(int, input().split())) # N 個の円盤の半径を記録するリスト
maxr = 0
cnt = 0
for i in range(N - 1, -1, -1):
    if r[i] > maxr:
        cnt += 1
        maxr = r[i]
print(cnt)
